projects/vdk-plugins/vdk-ingest-http/tests/test_ingest_over_http.py

```diff
@@ -54,9 +58,62 @@ def test_ingest_over_http(mock_post):
     )
 
 
+@mock.patch("requests.post", side_effect=mocked_requests_post)
+def test_ingest_over_http_compression(mock_post):
+    threshold_bytes = 3
+    encoding = "utf-8"
+    job_context = MagicMock()
+    job_context.core_context.configuration.get_value.side_effect = [
+        threshold_bytes,
+        encoding,
+    ]
+    http_ingester: IngestOverHttp = IngestOverHttp(job_context)
+    http_ingester.ingest_payload(
+        payload=[payload],
+        destination_table="test_table",
+        target="http://example.com/data-source",
+    )
+
+    assert len(mock_post.call_args_list) == 1
```

Comment:
```suggestion
    assert mock_post.call_count == 1
```
nit: I think you can do it this way, although I'm not sure what the underlying implementation looks like; the two approaches might be identical.